File: backend-csharp/DTOs/Auth/LoginResponse.cs
namespace SousChefBackend.DTOs.Auth;

public class LoginResponse
{
    public long UserId { get; set; }
    
    // 🔥 [新增]
    public int KitchenId { get; set; } 
    
    public TokenInfo Token { get; set; } = new();
}

public class TokenInfo
{
    public string AccessToken { get; set; } = string.Empty;
    public int ExpiresIn { get; set; } = 3600; // 默认 1 小时
}

File: backend-java/src/main/java/com/souschef/dto/auth/LoginResponse.java
package com.souschef.dto.auth;

import lombok.Data;

@Data
public class LoginResponse {
    private Long userId;
    private Integer kitchenId;
    private TokenInfo token;
    
    @Data
    public static class TokenInfo {
        private String accessToken;
        private Integer expiresIn = 3600;
    }
}

